data-esnext.js

```diff
@@ -51,7 +51,7 @@ exports.tests = [
   spec: 'http://wiki.ecmascript.org/doku.php?id=strawman:do_expressions',
   exec: function () {/*
     return do {
-      let x = 23;
+      var x = 23;
```

Comment:
This must be a block-level variable to properly test block scope - in other words, it must be `let` or `const`.

Why do you think this is an important change to make? Does there exist any implementation that supports do expressions but not let/const?